python/tvm/relay/build_module.py

```diff
@@ -222,7 +225,8 @@ def build(func,
     cfg = BuildConfig.current
 
     with tophub_context:
-        func = optimize(func, params)
+        with target:
```

Comment:
May I ask why we need to have `with target` here? It seems that constant folding is not target dependent because it always uses llvm as the target.

Reply:
@zhiics alter_op_layout relies on current target to query autotvm log

Reply:
@vinx13 I just saw that. Thanks for your quick response. Can we pass the target to later layout? I am asking is because for heterogeneous compilation we will pass in multiple targets. It is probably not convenient to know which target to be `with` here.